scalpel-core/src/Text/HTML/Scalpel/Internal/Scrape.hs

```diff
@@ -22,6 +22,7 @@ import Text.HTML.Scalpel.Internal.Select.Types
 import Control.Applicative
 import Control.Monad
 import Data.Maybe
+import Text.StringLike (StringLike, castString)
```

Comment:
Nice! I never really liked that StringLike was a part of the TagSoup package.

Reply:
I think StringLike is an abomination as well